### PR TITLE
Alignment with DCAT-AP 3

### PR DESCRIPTION
**Pull request by andrea-perego**
The alignment with DCAT-AP 3 is limited to what follows:
- Support for data services.
- Properties used to specify start and end of a period.

Summary of changes:

- Added folder `docs/1.0` and copied there files of DCAT-AP-JRC v1.
- Added README file to folder `docs/`
- Added class `dcat:DataService` and related properties in the specification.
- Replaced `dctype:Service` with `dcat:DataService` for endpoints.
- Replaced `schema:startDate` with `dcat:startDate`.
- Replaced `schema:endDate` with `dcat:endDate`.
- Added section describing the scope of the revision in the specification.
- Added section on deprecated classes.
- Added section on deprecated properties for temporal coverage.
- Added changelog in the specification.
- Revised class diagram in `docs/img` to add class `dcat:DataService`.
- Revised `docs/config.js` for the new version; added entries for HYDRA, DCAT-AP-JRC-1, JRC-DP-2015, JRC-DP-2019, HVD.
- Added RDF file in `docs/examples` with an example of a `dcat:DataService`.
- Added section on describing how to upgrade metadata to the new version.
- Added file in `docs/sparql` with a `CONSTRUCT` SPARQL query to automatically generate a data service from a distribution available from it.
- Added file in `docs/sparql` with a `DELETE/INSERT` SPARQL query to replace all the instances of `dctype:Service` with instances of `dcat:DataService`.
- Added file in `docs/sparql` with a `DELETE/INSERT` SPARQL query to replace all the instances of `schema:startDate` with instances of `dcat:startDate`.
- Added file in `docs/sparql` with a `DELETE/INSERT` SPARQL query to replace all the instances of `schema:endDate` with instances of `dcat:endDate`.
- Revised main README file.
- Editorial revisions for the new version.

Preview: https://raw.githack.com/ec-jrc/dcat-ap-jrc/dev/docs/index.html

Diff: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fec-jrc.github.io%2Fdcat-ap-jrc%2F&doc2=https%3A%2F%2Fraw.githack.com%2Fec-jrc%2Fdcat-ap-jrc%2Fdev%2Fdocs%2Findex.html

### CODE REVIEW

File: README.md
Comment (H-a-g-L) [inline README.md:1]: ok

File: docs/config.js [hunk 1]
```diff
@@ -41,32 +41,43 @@ var respecConfig = {
...
+    previousURI:   "https://ec-jrc.github.io/dcat-ap-jrc/1.0/",
     latestVersion: "https://ec-jrc.github.io/dcat-ap-jrc/",
     edDraftURI:    "https://ec-jrc.github.io/dcat-ap-jrc/",
     issueBase:     "https://github.com/ec-jrc/dcat-ap-jrc/issues/",
     github:        "https://github.com/ec-jrc/dcat-ap-jrc/",
 //    overrideCopyright: '<p class="copyright"><a href="https://europa.eu/european-union/abouteuropa/legal_notices_en#copyright_notice">Copyright</a> © 2014-2019 <a href="https://europa.eu/">European Union</a>. This document is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by/4.0/" class="subfoot">Creative Commons Attribution 4.0 License</a>.</p>',
     editors: [
     {
-      name:       "Andrea Perego",
+      name:       "Hagar Lowenthal",
```
Comment (H-a-g-L) [inline docs/config.js:52]: Andrea should appear as lead editor (appear as 1st on list)

File: docs/config.js [hunk 2]
```diff
@@ -41,32 +41,43 @@ var respecConfig = {
...
+      name:       "Andrea Perego",
+      company:    "European Parliament",
+      orcid:      "0000-0001-9300-2694",
+      companyURL: "https://europarl.europa.eu/"
     }
     ],
     authors: [
     {
-      name:       "Andrea Perego",
+      name:       "Hagar Lowenthal",
```
Comment (H-a-g-L) [inline docs/config.js:65]: Andrea should appear as lead author (appear 1st on te list)

File: docs/1.0/examples/catalogue.rdf [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      The JRC Data Catalogue gives access to the multidisciplinary 
+      data produced and maintained by the Joint Research Centre, 
+      the European Commission's in-house science service providing 
+      independent scientific advice and support to policies of the 
+      European Union.
+    </dct:description>
+    <dct:publisher rdf:resource="http://publications.europa.eu/resource/authority/corporate-body/JRC"/>
+    <dcat:contactPoint>
+      <vcard:Kind>
+        <vcard:hasURL rdf:resource="https://data.jrc.ec.europa.eu/contact"/>
```
Comment (H-a-g-L) [inline docs/1.0/examples/catalogue.rdf:17]: updated URL: https://data.jrc.ec.europa.eu/about#contact

File: docs/1.0/examples/dataset.rdf [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+          2018-07-18
+        </dct:issued>
+      </adms:Identifier>
+    </adms:identifier>
+
+<!-- Temporal coverage -->
+
+    <dct:temporal>
+      <dct:PeriodOfTime>
+        <schema:startDate rdf:datatype="http://www.w3.org/2001/XMLSchema#date">
```
Comment (H-a-g-L) [inline docs/1.0/examples/dataset.rdf:87]: update to <dcat:startDate

File: docs/1.0/examples/dataset.rdf [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+          2015-12-01
+        </schema:startDate>
+        <schema:endDate rdf:datatype="http://www.w3.org/2001/XMLSchema#date">
```
Comment (H-a-g-L) [inline docs/1.0/examples/dataset.rdf:90]: update to <dcat:endDate

File: docs/1.0/examples/temporal-coverage.rdf [hunk 6]
```diff
@@ -0,0 +1,18 @@
+  <dcat:Dataset rdf:about="http://data.europa.eu/89h/1234567890">
+    ...
+
+<!-- Temporal coverage -->
+
+    <dct:temporal>
+      <dct:PeriodOfTime>
+        <schema:startDate rdf:datatype="http://www.w3.org/2001/XMLSchema#date">
```
Comment (H-a-g-L) [inline docs/1.0/examples/temporal-coverage.rdf:8]: update to <dcat:startDate

File: docs/1.0/examples/temporal-coverage.rdf [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+          2015-12-01
+        </schema:startDate>
+        <schema:endDate rdf:datatype="http://www.w3.org/2001/XMLSchema#date">
```
Comment (H-a-g-L) [inline docs/1.0/examples/temporal-coverage.rdf:11]: update to <dcat:endDate

File: docs/1.0/examples/endpoint.rdf [hunk 8]
```diff
@@ -0,0 +1,46 @@
+  <rdf:Description rdf:about="http://data.jrc.ec.europa.eu/endpoint/liscoast">
+    <rdf:type rdf:resource="http://www.w3.org/ns/dcat#Catalog"/>
+    <rdf:type rdf:resource="http://purl.org/dc/dcmitype/Service"/>
```
Comment (H-a-g-L) [inline docs/1.0/examples/endpoint.rdf:3]: change to: <rdf:type rdf:resource="http://www.w3.org/ns/dcat#DataService"/>

File: docs/1.0/index.html [hunk 9]
```diff
@@ -0,0 +1,12060 @@
...
+
+<section id="endpoint-supported-schema">
+
+<h2>Property: supported schema</h2>
+
+<table class="def">
+<tbody>
+<tr>
+<th>Label</th>
+<td>protocol</td>
```
Comment (H-a-g-L) [inline docs/1.0/index.html]: change to "metadata schema"

File: docs/1.0/index.html [hunk 10]
```diff
@@ -0,0 +1,12060 @@
...
+<th>Range</th>
+<td><a data-cite="DCTERMS#terms-PeriodOfTime"><code title="http://purl.org/dc/terms/PeriodOfTime">dct:PeriodOfTime</code></a> (<a class="no-sectionRef" href="#temporal-coverage"></a>)</td>
+</tr>
+<tr>
+<th>Definition</th>
+<td>The temporal coverage of the dataset.</td>
+</tr>
+<tr>
+<th>Usage note</th>
+<td>The interval MUST be specified as defined in [[DCAT-AP]], namely, by indicating the start and/or end date by using <a href="http://schema.org/startDate"><code title="http://schema.org/startDate">schema:startDate</code></a> and <a href="http://schema.org/endDate"><code title="http://schema.org/endDate">schema:endDate</code></a>, respectively.</td>
```
Comment (H-a-g-L) [inline docs/1.0/index.html:3178]: change from sdo:startDate and :endDate to dcat:startDate and :endDate

File: docs/1.0/index.html [hunk 11]
```diff
@@ -0,0 +1,12060 @@
...
+<tr>
+<th>Definition</th>
+<td>The contact point for the collection.</td>
+</tr>
+<tr>
+<th>Scope</th>
+<td><a class="scope" href="#metadata-scope">All</a></td>
+</tr>
+<tr>
+
```
Comment (H-a-g-L) [inline docs/1.0/index.html:9182]: add Data Service: "This class is meant to be used for resources whose relationship with the dataset are different from <a class="no-sectionRef" href="#dataset-distribution">distribution</a>, <a class="no-sectionRef" href="#dataset-publication">publication</a>, <a class="no-sectionRef" href="#data-service">data service</a>, <a class="no-sectionRef" href="#dataset-input-data">input data</a>, <a class="no-sectionRef" href="#dataset-lineage">lineage</a>, <a class="no-sectionRef" href="#dataset-usage-notes">usage notes</a>."

**Review by H-a-g-L — COMMENTED** [review]
Minor updates mainly for revision of examples to match the new version

File: docs/1.0/examples/catalogue.rdf (hunk 3 above)
Comment (andrea-perego) [reply docs/1.0/examples/catalogue.rdf:17]: Addressed by commit https://github.com/ec-jrc/dcat-ap-jrc/pull/2/commits/a26901c78c4aa2cc87d5992affd90e60d47950f4 in version 2:

https://github.com/ec-jrc/dcat-ap-jrc/blob/a26901c78c4aa2cc87d5992affd90e60d47950f4/docs/examples/catalogue.rdf#L14-L17

File: docs/1.0/examples/dataset.rdf (hunk 4 above)
Comment (andrea-perego) [reply docs/1.0/examples/dataset.rdf:87]: Addressed by commit https://github.com/ec-jrc/dcat-ap-jrc/pull/2/commits/0769761fee10454165f94484602832775777f9d1 in version 2:

https://github.com/ec-jrc/dcat-ap-jrc/blob/0769761fee10454165f94484602832775777f9d1/docs/examples/dataset.rdf#L84-L87

File: docs/1.0/examples/dataset.rdf (hunk 5 above)
Comment (andrea-perego) [reply docs/1.0/examples/dataset.rdf:90]: Addressed by commit https://github.com/ec-jrc/dcat-ap-jrc/pull/2/commits/0769761fee10454165f94484602832775777f9d1 in version 2:

https://github.com/ec-jrc/dcat-ap-jrc/blob/0769761fee10454165f94484602832775777f9d1/docs/examples/dataset.rdf#L87-L90

File: docs/1.0/examples/temporal-coverage.rdf (hunk 6 above)
Comment (andrea-perego) [reply docs/1.0/examples/temporal-coverage.rdf:8]: Addressed by commit https://github.com/ec-jrc/dcat-ap-jrc/pull/2/commits/0769761fee10454165f94484602832775777f9d1 in version 2:

https://github.com/ec-jrc/dcat-ap-jrc/blob/0769761fee10454165f94484602832775777f9d1/docs/examples/temporal-coverage.rdf#L5-L8

File: docs/1.0/examples/temporal-coverage.rdf (hunk 7 above)
Comment (andrea-perego) [reply docs/1.0/examples/temporal-coverage.rdf:11]: Addressed by commit https://github.com/ec-jrc/dcat-ap-jrc/pull/2/commits/0769761fee10454165f94484602832775777f9d1 in version 2:

https://github.com/ec-jrc/dcat-ap-jrc/blob/0769761fee10454165f94484602832775777f9d1/docs/examples/temporal-coverage.rdf#L8-L11

File: docs/1.0/examples/endpoint.rdf (hunk 8 above)
Comment (andrea-perego) [reply docs/1.0/examples/endpoint.rdf:3]: Addressed by commit https://github.com/ec-jrc/dcat-ap-jrc/pull/2/commits/c4b800d58e557530c35e15586eef4f5cc131152d in version 2:

https://github.com/ec-jrc/dcat-ap-jrc/blob/c4b800d58e557530c35e15586eef4f5cc131152d/docs/examples/endpoint.rdf#L1-L3

File: docs/1.0/index.html (hunk 9 above)
Comment (andrea-perego) [reply docs/1.0/index.html]: Addressed by commit https://github.com/ec-jrc/dcat-ap-jrc/pull/2/commits/82068f0e871bd4eecfc059d319731d8ae06a1496 :

https://github.com/ec-jrc/dcat-ap-jrc/blob/82068f0e871bd4eecfc059d319731d8ae06a1496/docs/1.0/index.html#L7510-L7513

File: docs/1.0/index.html (hunk 10 above)
Comment (andrea-perego) [reply docs/1.0/index.html:3178]: Addressed by commit https://github.com/ec-jrc/dcat-ap-jrc/pull/2/commits/a95c579945ad98f4b78de7255ebc06f2037a329b in version 2:

https://github.com/ec-jrc/dcat-ap-jrc/blob/a95c579945ad98f4b78de7255ebc06f2037a329b/docs/index.html#L3256

File: docs/1.0/index.html (hunk 11 above)
Comment (andrea-perego) [reply docs/1.0/index.html:9182]: Addressed by commit https://github.com/ec-jrc/dcat-ap-jrc/pull/2/commits/91827c64424228d9102f8fbf771b74bad0608a19 in version 2:

https://github.com/ec-jrc/dcat-ap-jrc/blob/91827c64424228d9102f8fbf771b74bad0608a19/docs/index.html#L10089

**Review by H-a-g-L — APPROVED** [review]

**Pull request merged**